addon/components/project-selector/template.hbs

```diff
@@ -85,13 +85,21 @@
         {{/if}}
         {{#if (or willCreateComponent willMoveToNode)}}
             <p class="m-t-lg">
-                {{t "eosf.components.moveToProject.convertOrCopyMessage"}}
-            </p>
-            <p class="text-danger m-t-sm">
-                {{#unless isProjectPublic}}
-                    {{t "eosf.components.moveToProject.noLongerPublicWarning"}}
-                {{/unless}}
+                {{#if isChildNode}}
+                    {{t "eosf.components.moveToProject.convertOrCopyMessage.component"}}
+                {{else}}
+                    {{t "eosf.components.moveToProject.convertOrCopyMessage.project"}}
+                {{/if}}
             </p>
+            {{#unless isProjectPublic}}
+                <p class="text-danger m-t-sm">
+                    {{#if isChildNode}}
+                        {{t "eosf.components.moveToProject.noLongerPublicWarning.component"}}
+                    {{else}}
+                        {{t "eosf.components.moveToProject.noLongerPublicWarning.project"}}
+                    {{/if}}
```

Comment:
One of these will always display once the user has selected where to move the file and the destination is (or will be) private.